testing/simapp/test_helpers.go

```diff
@@ -113,6 +114,98 @@ func SetupWithGenesisValSet(t *testing.T, valSet *cmttypes.ValidatorSet, genAccs
 	return app
 }
 
+func setup(tb testing.TB, chainID string, withGenesis bool, invCheckPeriod uint) (*SimApp, GenesisState) {
+	tb.Helper()
+	db := dbm.NewMemDB()
```

Comment:
Those testing functions are from the current `initSetup`, I added some app options to run statesync with the testing app.